oidc-stub/src/MockOidcHandler.ts

```diff
@@ -0,0 +1,80 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { ResourcesEnum } from "./models/enums/ResourcesEnum";
+
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { AppError } from "./utils/AppError";
+import { OidcProcessor } from "./services/OidcProcessor";
+
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.F2F_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.AUTHORIZATIONCODE_LOGGER_SVC_NAME;
+
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
+
+class MockOidcHandler implements LambdaInterface {
+
+	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
+	async handler(event: APIGatewayProxyEvent, context: any): Promise<APIGatewayProxyResult> {
+		 switch (event.resource) {
+			case ResourcesEnum.OIDC_TOKEN:
```

Comment:
Howcomes you decided to have single handler handling multiple endpoints rather than just have different handlers per each endpoint?

Reply:
Was working on a POC and was easier to handle it all in a handler. I can separate it out if thats more preferable?

Reply:
Worked on getting them separated out.